view/frontend/web/js/autocomplete.js

```diff
@@ -13,6 +15,8 @@ define([
 ], function (
     $,
     algoliaBundle,
+    algoliaSearch,
+    algoliaAutocomplete,
```

Comment:
```suggestion
    autocomplete,
```

Reply:
Make sense. Keeping object name smaller have much benefit. Will fix it in a few.